answer/templates.html

```diff
@@ -17,6 +17,10 @@
 </script>
 <!-- runtimeFilters -->
 <script type="text/template" id="runtimeFilters">
-     runtimeFilters: [],
+     runtimeFilters: [{
+          columnName: 'color',
+          operator: RuntimeFilterOp.EQ,
```

Comment:
Is `RuntimeFilterOp` already part of imports?

Reply:
yes